docs/UserDefinedLanguageServer.md

```diff
@@ -88,7 +88,13 @@ level Trace used in LSP console.
 
 ### Using template
 
-The `Template combo-box` provides some `language servers templates`, pre-filled with server name, command, mappings and potential configuration. 
+The `Template file selector` can be used to select a directory from the file system to use a pre-existing `language server template`, 
+these templates are pre-filled with server name, command, mappings and potential configuration.
+
+LSP4IJ provided templates are located in `src/main/java/resources/templates`, 

```

Comment:
By default, LSP4IJ provided templates located in [templates](https://github.com/redhat-developer/lsp4ij/tree/main/src/main/resources/templates) directory classpath.

Reply:
Damn it, we should load the default templates from the jar at runtime?

Accessing resources as virtual files does not seem to be possible, at least I didn't find a clean way to do this.

I wonder if it would be better to still keep the template-ls.json for default templates and keep the file selector for loading custom templates.

Or alternatively copying the resource files to the file system and using the existing folder loading to pre load the template files.

Either way, there would be two different selectors on the UI. A combo-box for default templates and a dir selector for custom templates

Reply:
> Damn it, we should load the default templates from the jar at runtime?

It works like this today.

Reply:
Yeah, it's just not loading the template file as a VirtualFile, but as so often, right after complaining about I think I found a way.

It is a bit ugly though, I'll do some testing

It is still fine, that there are two selectors, one for defaults and another one for custom templates?

Reply:
This also resolves the question about the telemetry, we can just keep the original implementation and only change the way the templates are loaded

Reply:
I'll need to do some cleaning up, removing the old loading logic and making the UI a bit clearer. I at you once I've completed this